### PR TITLE
First hello

### PR DESCRIPTION
**Pull request by Cabraham1**
first microverse task with pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,15 +42,9 @@ To get a local copy up and running follow these simple example steps.
 
 👤 **Author1**
```
Comment (ShadyShawkat) [inline README.md:44]: - [ ] [OPTIONAL] Please write your name here instead of `Author1`.

File: README.md [hunk 2]
```diff
@@ -1,25 +1,25 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse
 
 > Description the project.
```
Comment (ShadyShawkat) [inline README.md:6]: - [ ] [OPTIONAL] Kindly make sure to add a meaningful description to your README.

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Cabraham1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Cabraham1) [reply README.md:6]: @ShadyShawkat thank you very much i will work on the README

**Pull request merged**